sdk/communication/azure-communication-sms/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0b7 (Unreleased)
+## 1.0.0 (2021-03-29)
```

Comment:
This will need a comment otherwise release will fail.. we typically use Update <package-name> version.
There should be some examples..